avocado/utils/disk.py

```diff
@@ -40,6 +40,65 @@ def get_disk_blocksize(path):
     return fs_stats.f_bsize
 
 
+def create_loop_device(size, blocksize=4096, directory=''):
+    """
+    Creates a loop device of size and blocksize specified.
+
+    :param size: Size of loop device, in bytes
+    :type syze: int
+    :param blocksize: block size of loop device, in bytes. Defaults to 4096
+    :type blocksize: int
+    :param directory: Directory where the backing file will be created.
+                      Defaults to current directory.
+    :type directory: str
+
+    :return: loop device name, if created. '' otherwise.
+    :rtype: str
+    """
+    cmd = "losetup --find"
+    loop = process.run(cmd, ignore_status=True,
+                       sudo=True).stdout_text.strip('\n')
+
+    loop_file = os.path.join(directory,
+                             "tmp_%s.img" % loop.split('/')[-1])
+    cmd = "dd if=/dev/zero of=%s" % loop_file
+    cmd += " bs=%d" % blocksize
+    cmd += " count=%d" % int(size / blocksize)
```

Comment:
Maybe, `cmd = "dd if=/dev/zero of={} bs={} cound={}".format(loop_file, blocksize, int(size/blocksize)"` (You can break lines if it is too long.